CMakeLists.txt

```diff
@@ -451,29 +448,51 @@ else()
     util/io_posix.cc)
 endif()
 
+if(WIN32)
+  set(SYSTEM_LIBS ${SYSTEM_LIBS} Shlwapi.lib Rpcrt4.lib)
+  set(ROCKSDB_STATIC_LIB rocksdblib${ARTIFACT_SUFFIX})
+  set(ROCKSDB_SHARED_LIB ${ROCKSDB_STATIC_LIB})
+  set(ROCKSDB_IMPORT_LIB rocksdb${ARTIFACT_SUFFIX})
```

Comment:
Why do you need import lib on windows? You just say link to the shared (DLL) and CMAKe knows that it needs to put the import lib there